src/frontend/hooks/server/maps.ts

```diff
@@ -34,6 +34,11 @@ export function useMapStyleJsonUrl() {
   return useQuery({
     queryKey: [MAPS_QUERY_KEY, 'stylejson-url', refreshToken],
     queryFn: async () => {
+      // If we're running E2E tests (e.g. on BrowserStack), fall back to a
+      // public Mapbox style rather than our local style server to avoid 502 errors.
```

Comment:
would add a link to the github issue in this comment for more direct access to additional details

```suggestion
      // If we're running E2E tests (e.g. on BrowserStack), fall back to a
      // public Mapbox style rather than our local style server to avoid 502 errors.
      // (see https://github.com/digidem/comapeo-mobile/issues/1008)
```